app/javascript/article-form/articleForm.jsx

```diff
@@ -312,6 +312,7 @@ export class ArticleForm extends Component {
         onSubmit={this.onSubmit}
         onInput={this.toggleEdit}
         data-testid="article-form"
+        aria-label="Edit an article"
```

Comment:
Added a form label so we can retrieve this by [form role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Form_Role) with the label `"Edit an article"`

Reply:
```suggestion
        aria-label="Edit post"
```

nitpick (not blocking) - The terminology we use in the UI is 'post' so it would be good to keep it consistent